3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -15,12 +15,18 @@ const rollDice = require('../../helpers/pokerDiceRoller');
 function rollTheDices() {
   // TODO Refactor this function
   const dices = [1, 2, 3, 4, 5];
-  return rollDice(1);
+  const combinedPromise = dices.map((element) => rollDice(element));
```

Comment:
This name is misleading. It is not a single promise but an array of promises. You should therefore choose a name that is a plural, e.g. `combinedPromises` or simply `promises`.

Reply:
The name is changed.